dashboard/app/views/admin_reports/funometer.html.haml

```diff
@@ -1,100 +1,77 @@
 - content_for :head do
   %script{src:'https://www.google.com/jsapi'}
 
-%h1
-  Fun-O-Meter Stats
+- if @stats.present?
+  %h1
+    Fun-O-Meter Stats
 
-%h2
-  Overall Percentage:
-  %span= number_to_percentage(@overall_percentage, precision: 2)
+  %h2
+    Overall Percentage:
+    %span= number_to_percentage(@stats['overall'], precision: 2)
 
-%br
+  %br
 
-= render 'funometer_percentages_by_day', :percentages_by_day => percentages_by_day
+  -# Renable the graphs after reconciling the different object types.
```

Comment:
my only real thought is that keeping things different is always a little strange. In particular, since some of the dashboards are real-time and this one is not, I'd be worried about some confusion. Would it be easy to stick a header somewhere on this dashboard explaining that this data is up to a day old?